tools/run_tests/dockerize/docker_run.sh

```diff
@@ -13,10 +13,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-# This script is invoked by build_docker_* inside a docker
+# This script is invoked by build_and_run_docker.sh inside a docker
 # container. You should never need to call this script on your own.
 
-set -ex
+set -e
+
+# TODO(jtattermusch): added in https://github.com/grpc/grpc/pull/17303, should be removed.
+export CONFIG=${config:-opt}
```

Comment:
What's the blocker here?

Reply:
I wanted to remove this separately in a followup PR. There might be some odd cornercase that relies on this (I'm not aware of one though).